voxxed-microservices-2018/module-01.adoc

```diff
@@ -15,73 +15,110 @@ These tasks typically require direct access to the cluster nodes and can easily
 A Kubernetes Operator is an application-specific controller that extends the Kubernetes API to create, configure, and manage instances of complex stateful applications on behalf of a Kubernetes user.
-It builds upon the basic Kubernetes resource and controller concepts but includes domain or application-specific knowledge to automate common tasks.
+It builds upon the basic Kubernetes resource and operator concepts but includes domain or application-specific knowledge to automate common tasks.
 
-Instead of relying on direct deployment and management of Zookeeper and Kafka clusters, Strimzi consists of a couple of these domain-specific controllers that monitor the state of the cluster, making adjustments in accordance to a desired state read from dedicated ConfigMaps.
+Instead of relying on direct deployment and management of Zookeeper and Kafka clusters, Strimzi consists of a couple of these domain-specific operators that monitor the state of the cluster, making adjustments in accordance to a desired state read from dedicated configuration resources.
 
-For creating an Apache Kafka cluster, for instance, you need to create a ConfigMap that describes the properties of the cluster, and the *_cluster controller_* will deploy the cluster for you.
-If you need to change the state of the cluster, for example for changing properties or for adding new instances, all you have to do is to modify the ConfigMap and the changes will be rolled out accordingly.
+For creating an Apache Kafka cluster, for instance, you need to create a resource of type `kafka` that describes the properties of the cluster, and the *_cluster operator_* will deploy the cluster for you.
+If you need to change the state of the cluster, for example for changing properties or for adding new instances, all you have to do is to modify the resource and the changes will be rolled out accordingly.
 
-Topic management works in a similar fashion: for creating and modifying topics, you only need to create and edit a set of ConfigMaps and the *_topic controller_* will do the work for you.
+Topic management works in a similar fashion: for creating and modifying topics, you only need to create and edit a set of resources of type `kafkatopic` and the *_topic operator* will do the work for you.
 
 You will do all this as part of the first lab.
 
+=== Connecting to Your VM
+
+First, connect via SSH to the Vagrant VM on your machine or your remote cloud environment.
+For a local Vagrant box run:
+
+[source, sh]
+$ vagrant ssh
+
+Otherwise, run:
+
+[source, sh]
+$ ssh <YOUR IP>
+
+If you've set up the environment using the provided Ansible scripts, switch to the `build` user:
+
+[source, sh]
+sudo su build
+
 === Installing Strimzi
 
-First, connect via SSH to the workstation.
-The keys for connecting remotely to the workstation are already provisioned on your machine.
+Start up OpenShift:
 
 [source, sh]
-$ ssh workstation-<GUID>.rhpds.opentlc.com
+$ oc cluster up --public-hostname=<YOUR IP>
```

Comment:
This can be misleading - <YOUR_IP> IP address of the Vagrant/cloud box or the laptop?